### ISSUE TITLE
Sockets

### ISSUE THREAD

**Issue by arsen3d** [dvasanth/portforward-over-libp2p#2]
The code appears to support socket streams, but the readme does not reference this. Just http/https.
https://github.com/dvasanth/portforward-over-libp2p/blob/main/README.md
Could you update readme to confirm this.

**Comment by dvasanth** [comment]
yes, its supported for port forwarding any socket application. Updated the readme.
